gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagement.java

```diff
@@ -27,5 +27,13 @@
  */
 public interface DagManagement {
 
+  /**
+   * Used to add a dagAction event to DagManagement
+   */
   void addDagAction(DagActionStore.DagAction dagAction) throws IOException;
+
+  /**
+   * Used to add reminder dagActions to the queue that already contain an eventTimestamp from the previous lease attempt
+   */
+  void addDagAction(DagActionStore.LeaseObject reminderDagActionLeaseObject) throws IOException;
```

Comment:
Shouldn't it be named just `dagActionLeaseObject`

Reply:
first time we pass `LeaseObject` to the MALA it's not a reminder and the value is false. Renamed metrics and the object